posts/2022/02/what-do-we-need-to-write.md

```diff
@@ -0,0 +1,126 @@
+that feeling when writing.
+- I don't have time to reflect on my work and record it.
+- It's difficult to translate non-linear thinking into a linear format 
+(writing).
+
+### Avoiding diminishing returns
+
+Now let's pretend we've navigated around the above obstacles and have something 
+(however messy) on the page. In most cases, that is only the first step. 
+
+When you are maintaining projects, juggling your community commitments, and 
+maybe even already writing documentation, how do you make good use of your 
+time while writing? How do you get it from *started* to *finished*? This was 
+another theme our team discussed, which can be distilled into the following:
+
+- The hardest part of writing is finishing it (especially when writing something over 
+a long period of time).
+- I can't afford to spend 6 hours on one sentence (but can easily  end up doing so).
+- Being responsible for what you say on a company blog can be stressful.
+- If I haven't looked at my drafts in a while, I don't usually find anything 
+there that I feel is worth continuing on.
+
+### Maybe it's not so bad
+
+Some of the authors' favorite parts of these lists is that many of the 
+obstacles complement each other! Just like there isn't one single kind of 
+author, there isn't one single set of needs people have in order to write. Do 
+we need people or do we need alone time? Do we need the force of deadlines or 
+the relaxation of an open-ended timeline? One of our team members even said
+
+> It's interesting to see how everyone is struggling in different ways.
+
+Bring those thoughts with you while we discuss the next question…
+
+## How do people like to write?
+
+Sometimes writings do successfully get published! 🎉 We asked our team about 
+what goes right when writing does get completed as planned. People 
+commented that writing works best for them when the following happens:
+
+- Knowing who you are writing for is important. For practice, folks suggested 
+writing for yourself or close allies.
+- Writing about topics you like (or at least tying another topic into that).
+- I like to write as part of a organized meet up where you critique and share 
+each other's work with the intent of publishing at the end. ([Writer's workshop style](https://github.com/Quansight/writers-workshop))
+- Having a good editor / someone to provide constructive critiques is so helpful.
+- Co-authoring is the best!
+- As a way to bring people into new projects.
+- It's so helpful to be hyped and have that support system.
+- To by hyped by others in the form of "I double-doggy-dare you" or "I bet a 
+dollar you can."
+
+The "I published a blog post in the last few months" team also provided some 
+writing lifestyle advice:
+
+- Write in a cabin in the woods (warning: you may risk hiking instead of 
+writing)
+- Drink tea while writing.
+- A legal pad and a computer go together. No computer writing alone.
+- Relax while writing. You'll think better. (When there's no deadline!)
+
```

Comment:
After reading about the tyranny of the blank page, I'd like some specific advice to getting up and running when facing that tyrant.  Here is a suggestion for some text to add:

"Most people agreed that to overcome the tyranny of the blank page, it's helpful to just write.  Write anything.  It can be nonsense, gibberish, garbage, or downright silly, but just write. 

If that's not effective, here are some other ideas:
- Try to sneak up on the empty page by doing something you enjoy that is related to what you want to write.  For example, if you love to draw, try drawing something related to your topic and let it pull you into that world.
- Try open-ended questions, like 'What message do I want the reader to learn?' or 'What's the coolest thing about this topic?'
- Pretend you're having a conversation with someone.  How would you start describing the topic to them?
- (more)"

I can't remember all of the ideas, so these are mine.  I'd like to see others here too.  Feel free to reject some of mine, change the order, or whatever.